### PR TITLE
[Docs] typos and such

### PR DESCRIPTION
**Pull request by Green-Sky**
Found some mistakes in the docs.

disclaimer: I'm not a native speaker either...

### CODE REVIEW

**Review by AlexDenisov — APPROVED** [review]

File: docs/OpenSSLTutorial.rst [hunk 1]
```diff
@@ -56,7 +56,7 @@ If you run it on Linux you should see the following warnings:
 In order to run tests (original or mutated) Mull needs to feed the whole program and all its dependencies into the JIT engine.
 This includes dynamic libraries. Mull extracts them from the executable. On macOS, in most of the cases, the dynamic
 libraries have the full path. However, on Linux there are only names of the libraries, while the full paths normally determined
-by the dynamic linker ``ldd``. Mull does not assume any of the paths, so this is responsibility of the user to provide the paths.
```
[the remark this answers is not in the document]
Comment (Green-Sky) [reply docs/OpenSSLTutorial.rst:59]: yea, i think you are right. `ld` would be the linker used at "compile time".
shall i change it to `ld.so` and rebase?

[the remark this answers is not in the document]
Comment (AlexDenisov) [reply docs/OpenSSLTutorial.rst:59]: Yeah, that would be great!

**Comment by Green-Sky** [comment]
should be good

**Comment by Green-Sky** [comment]
btw, i got to know about this project via this years llvm dev meeting video https://www.youtube.com/watch?v=DfoS9kdTWmI and already like the tool :heart:

**Pull request merged**